index.js

```diff
@@ -0,0 +1,34 @@
+var fs = require('fs'),
+    path = require('path'),
+    extend = require('extend');
+
+function load() {
+    // Recursively load one or more directories passed as arguments.
+
+    var dir, result = {};
+
+    function processFilename(fn) {
+        var fp = path.join(dir, fn),
+            // If the given filename is a directory, recursively load it.
+            extra = fs.statSync(fp).isDirectory() ? load(fp) : require(fp);
+
+        // The JSON data is independent of the actual file
+        // hierarchy, so it is essential to extend "deeply".
+        result = extend(true, result, extra);
+    }
+
+    for (dir of arguments) {
+        dir = path.resolve(dir);
+        fs.readdirSync(dir).forEach(processFilename);
+    }
+
+    return result;
+}
+
+module.exports = load(
```

Comment:
Lets comment out api, http, and javascript. So that if we release now, these won't be exported. (I'm thinking about releasing a pre version, so that you can get started on loading this into Kumascript :))